apps-rendering/src/bodyElement.ts

```diff
@@ -110,6 +110,18 @@ interface AudioAtom {
 	title: string;
 }
 
+interface NewsletterSignUp {
+	kind: ElementKind.NewsletterSignUp;
+	newsletter: {
+		id: string;
+		displayName: string;
+		frequency: string;
+		description: string;
+		group: string;
+		theme: string;
```

Comment:
Is `theme` equivalent to `ArticleTheme` in the `Format` type, or does it just happen to have the same name?

Reply:
I think it was intended to be equivalent to `ArticleTheme`, but this isn't formalised in the codebase for the newsletters API - so the value should be an `ArticleTheme`, but it wouldn't be safe to type it as one rather than as a `string`

Reply:
Makes sense. In that case I think we can narrow the type when we parse the data coming from MAPI? That way we can give the `theme` field here in this body element a more accurate type. A function something like:

```ts
const themeFromString = (theme: string): ArticleTheme => {
  switch (theme) {
    case 'opinion':
      return ArticlePillar.Opinion;
    ...
    case 'news':
    default:
      return ArticlePillar.News;
  }
}
```

This is very similar to how we derive the main theme field in `Format`; we take the pillar `string` CAPI gives us and turn it into an `ArticleTheme`. In fact we can probably make that function a bit more generic so we don't duplicate the logic.

Reply:
sounds good - using the ArticleTheme enum instead of a string now!